packages/influx-metrics-tracker/src/helpers/exponential-backoff.spec.ts

```diff
@@ -0,0 +1,51 @@
+import * as exponentialBackoff from './exponential-backoff';
+
+const mockFunction = jest.fn();
+jest.useFakeTimers();
+jest.spyOn(global, 'setTimeout');
+
+function setMockToThrowErrorNTimes(times: number) {
+  for (let i = 0; i < times; i++) {
+    mockFunction.mockImplementationOnce(() => {
+      throw Error();
+    });
+  }
+}
+
+describe('executeCallbackOrExponentiallyBackOff', () => {
```

Comment:
Capitalize E for consistency

Reply:
This is the function name which is why I've left it as lowercase